### DISCUSSION TITLE
api key question

### DISCUSSION THREAD

**Discussion by efi20232** [AntonOsika/gpt-engineer#432] [General]
do i need pay for chatgpt4 api usage/month in order to make my api key work here?
cuz i am getting this error

(gpt-eng) PS C:\gpt-engineer\gpt_engineer> python main.py example
INFO:openai:error_code=model_not_found error_message="The model 'gpt-4' does not exist" error_param=model error_type=invalid_request_error message='OpenAI API error received' stream_error=False
Model gpt-4 not available for provided API key. Reverting to gpt-3.5-turbo. Sign up for the GPT-4 wait list here: https://openai.com/waitlist/gpt-4-api

**Comment by krclarke101** [comment]
I am getting the same, but I was not sure if I'd set something up incorrectly or if it has to be a GPT-4 key. I know the setup guides says 4.
